meta-mbl-distro/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -42,6 +42,38 @@ python __anonymous() {
         d.appendVar("FSTAB_LINES", fstab_line)
 }
 
+# Write the partition sizes and offsets to files which will be placed in /config/factory
+python do_save_partition_info() {
+    import pathlib
+
+    path_prefix = d.getVar("WORKDIR")
```

Comment:
Can you `s/path_prefix/workdir`? I think it is more clear reading `workdir` and understand what is going on. And looking at the oe-core classes they follow this naming convention.